src/data/ExplorePageData.ts

```diff
@@ -0,0 +1,29 @@
+import { Theme } from '@/theme/Theme';
+import { GreaterMattapanZoning as GreaterMattapanZoningData } from '@/data/GreaterMattapanZoning';
+import { BlueHillAveCorridor as BlueHillAveCorridorData } from '@/data/BlueHillAveCorridor';
+
+export const ExplorePageData: MapGeoJsonData[] = [
+  {
+    data: BlueHillAveCorridorData,
+    id: 'blueHillAveCorridor',
+    name: 'Blue Hill Ave Corridor',
+    type: 'line',
+    color: Theme.colors.blue,
+    visible: true,
+  },
+  {
+    data: GreaterMattapanZoningData,
+    id: 'greaterMattapanZoning',
+    name: 'Greater Mattapan Zoning',
+    type: 'line',
+    color: Theme.colors.orange,
+    visible: true,
+  },
+];
```

Comment:
So basically to add more data points, just fill out another `MapGeoJsonData`-like object in this array

Reply:
As we spoke about, we might want to make this its own file if we go with some github action based approaches to update it, or house this piece elsewhere like S3 or whatever.